app/com/lynxanalytics/biggraph/controllers/Workspace.scala

```diff
@@ -398,7 +399,7 @@ object BoxOutputState {
     BoxOutputState(BoxOutputKind.Plot, Some(plot))
   }
 
-  def from(
+  def exportResult(
```

Comment:
This couldn't stay `from()` because we have two outputs that are `Scalar[String]` — export results and HTMLs. Hence the renaming.